recipes/rapidjson/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "1.1.0" %}
+
+package:
+  name: rapidjson
+  version: {{ version }}
+
+source:
+  url: https://github.com/miloyip/rapidjson/archive/v{{ version }}.tar.gz
+  fn: {{ version }}.tar.gz
+  md5: badd12c511e081fec6c89c43a7027bce
+
+build:
+  number: 1
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+
+test:
+  commands:
+    - test -e $PREFIX/include/rapidjson/rapidjson.h  # [unix]
+    - if exist %LIBRARY_INC%\rapidjson\rapidjson.h (exit 0) else (exit 1)  # [win]
```

Comment:
Could we add a test for the presence of the `RapidJSONConfig.cmake` and `RapidJSONConfigVersion.cmake`?

The RapidJSONcmake install places these files in directory which is `PREFIX/lib/cmake/RapidJSON` which is quite unusual but resolved correctly by cmake.